### ISSUE TITLE
Error: No valid session available from this socket.

### ISSUE THREAD

**Issue by LA-Taylor** [balderdashy/sails#1927]
Hi,

I'm wondering if someone may please provide some insight/assistance. I'm trying to configure my angular application to talk to a seperate sails.js backend via socket.io. (CORS is enabled and I have turned off authentication for now in sockets.js).

Everytime I try to initiate a connection, I get the following from sails.js: 

info: handshake authorized kCuP03bnPesq61amoThC
error: Error: No valid session available from this socket.

Can someone please how to resolve this and why this is happening? I've read about sending a jsonp request, but the aim of the sails.js application is to act as a service via REST and therefore should be stateless.

Thanks.

**Comment by sailsbot** [comment]
Thanks for posting, @LA-Taylor.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
